tsconfig.json

```diff
@@ -22,6 +22,12 @@
     "incremental": false,
     "typeRoots": ["node_modules/@types", "src/types"]
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "generators/story.js",
+    "plopfile.js"
```

Comment:
Is it possible to do it with typescript files ?

Reply:
I tried (and @ivan-dalmet can agree) plop is not ready enough for TypeScript.